nemo/collections/asr/parts/utils/manifest_utils.py

```diff
@@ -368,6 +369,8 @@ def read_manifest(manifest: str) -> List[dict]:
     Returns:
         data (list): List of JSON items
     """
+    manifest = StoreObject(manifest).get()
```

Comment:
This seems a bit strange to me as documentation of `StoreObject` mentions that it's used for AIS objects, but we use it here and in other files for both ais and regular objects. Maybe we should update documentation or maybe change api, such that it's more visible (e.g., adding `get_manifest` or `prepare_manifest` function that will wrap this)?

Reply:
It's also a bit strange that in some cases we use "functional" api, like `store_path_to_local_path` or `is_store_path` and in some cases we use "class" api like here. Would be cleaner in my opinion if it was the same style (e.g., always wrapping with class objects and then using methods or always using top-level functions)